README.md

```diff
@@ -1,4 +1,5 @@
 # load_confounds
+Adding acompcor compartment selection and 50% variance.
```

Comment:
Please remove. I will add contributions with the all-contributors bot, and the release change logs. This contribution will not be forgotten :)